app/models/manageiq/providers/ibm_power_hmc/infra_manager/provision/cloning.rb

```diff
@@ -3,11 +3,12 @@ def clone_complete?
     $ibm_power_hmc_log.info("#{self.class}##{__method__}")
     raise MiqException::MiqProvisionError, "VM capture to template failed" if phase_context[:new_vm_ems_ref].nil?
 
-    if source.template?
-      association = :vms
-    else
-      association = :miq_templates
-    end
+    association =
+      if source.template?
+        :vms
+      else
+        :miq_templates
+      end
```

Comment:
For something this simple we usually prefer to use the ternary operator

```suggestion
    association = source.template? ? :vms : :miq_templates
```